download/releaseNotes/releaseNotes8.adoc

```diff
@@ -0,0 +1,17 @@
+= Release Notes 8
+:awestruct-description: New and noteworthy, demos and status for OptaPlanner 7.
+:awestruct-layout: releaseNotesBase
+:awestruct-priority: 1.0
+:awestruct-release_notes_version: 8
+:awestruct-release_notes_version_qualifier: Final
+
+== New and noteworthy: Engine 8.0.0.Final
+
+=== XML Schema Definition for both solverConfig.xml and benchmarkConfig.xml
```

Comment:
I'd somehow try to incorporate the fact that XSDs are optional. If you use them, you'll have to change your XML, possibly significantly. If you don't use it, nothing changes for you and you don't get IDE support.

Reply:
I agree. Although @ge0ffrey wanted to make a strong nudge to users to migrate to the XSD.